### PR TITLE
Feature: Add username to user's model

### PR DESCRIPTION
**Pull request by xenirio**

### CODE REVIEW

File: config/backend.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
 
   db:
-    image: mysql:5.7
+    image: mysql:8.0
```
Comment (chumaknadya) [inline config/backend.yml]: I don't think we need to change mysql image in this PR

**Comment by chumaknadya** [comment]
Also be sure you don't have private and public keys in this PR

File: config/backend.yml (hunk 1 above)
Comment (ghost) [reply config/backend.yml]: Okay
I reverted this change

File: db/migrate/20210316083841_add_username_to_users.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class AddUsernameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :username, :string, null: true
```
Comment (chumaknadya) [inline db/migrate/20210316083841_add_username_to_users.rb]: username should be after uid field

``add_column :users, :username, :string, null: true, after: :uid``

**Review by chumaknadya — CHANGES_REQUESTED** [review]

**Review by chumaknadya — APPROVED** [review]

**Pull request merged**